sharding-core/sharding-core-common/src/test/java/org/apache/shardingsphere/core/strategy/keygen/SnowflakeShardingKeyGeneratorTest.java

```diff
@@ -149,6 +164,24 @@ public void assertSetWorkerIdFailureWhenNegative() {
         keyGenerator.generateKey();
     }
     
+    @Test(expected = IllegalArgumentException.class)
+    public void assertSetMaxVibrationOffsetFailureWhenNegative() {
+        SnowflakeShardingKeyGenerator keyGenerator = new SnowflakeShardingKeyGenerator();
+        Properties properties = new Properties();
+        properties.setProperty("max.vibration.offset", String.valueOf(-1));
+        keyGenerator.setProperties(properties);
+        keyGenerator.generateKey();
+    }
+    
+    @Test(expected = IllegalArgumentException.class)
+    public void assertSetMaxVibrationOffsetFailureWhenZero() {
```

Comment:
zero of vibration offset should be fine, which means every milliseconds they are start from zero, is it?